metrics/cgroups/v1/metrics.go

```diff
@@ -148,16 +148,12 @@ func (c *Collector) collect(entry entry, ch chan<- prometheus.Metric, block bool
 		return
 	}
 
-	data, err := typeurl.UnmarshalAny(stats)
-	if err != nil {
+	s := &v1.Metrics{}
```

Comment:
This chunk of code has remained the same in over 4 years now. I am not sure this is anything specific to containerd 1.7.5 like the github issue here mentions https://github.com/containerd/containerd/issues/9052 . What version were you previously on? Has there been any changes on your application?

Reply:
We are using containerd with kubelet, there's nothing that comes to mind. I'll run this test on a kubelet and let you know